hendrix/resources.py

```diff
@@ -215,9 +215,6 @@ def DjangoStaticResource(path, rel_url='static'):
     rel_url = rel_url.strip('/')
     StaticFilesResource = MediaResource(path)
     StaticFilesResource.namespace = rel_url
-    chalk.green(
```

Comment:
Roger.  Makes sense.

Maybe have an optionally separate log for statics altogether?